### DISCUSSION TITLE
[Question] Compatibility

### DISCUSSION THREAD

**Discussion by AlphaNecron** [amwx/FluentAvalonia#213] [Q&A]
Why FluentAvalonia refuses to run on Windows 10 or lower but working fine on MacOS and Linux? Can you explain that please?

**Comment by amwx** [comment]
I don't support running on Windows 7 or 8. While Win 7 was a great OS, it's out dated and I'm not interested in supporting an OS that came out 15 years ago (support also ends next year from MS too), and Win 8 was a piece of crap and is out of support anyway. 
In addition to my personal biases, I also use modern WinRT APIs for some things only available in or after Win 10 and don't want to pollute the code with version checks.

However, FA does work with Windows 10 so if you're having an issue please give more details on what the issue is and/or how to repro it so I can help. 

Also converting to discussion, which is better suited for this type of question

**Reply by AlphaNecron** [reply]
Oh nice, I'm making a launcher and wanting it to work on all platforms though. Thanks for the explanation.
